qa/smoke-test-multinode/src/yamlRestTest/resources/rest-api-spec/test/smoke_test_multinode/30_desired_balance.yml

```diff
@@ -30,6 +30,8 @@ setup:
   - gte: { stats.computation_time_in_millis: 0 }
   - gte: { stats.reconciliation_time_in_millis: 0 }
 
+  - is_true: 'cluster_balance_stats'
```

Comment:
I am surprised that we have something regarding `desired_balance` in `smoke_test_multinode`. Should we rely only on a dedicated spec below?

Reply:
I think (despite the comment below) we're relying on this being run in a multi-node cluster which isn't the case for the general `rest-api-spec` tests.